src/CLI/clitree/cli-xml/interface.xml

```diff
@@ -96,6 +96,20 @@ limitations under the License.
    </ACTION>
    </COMMAND>
 
+   <COMMAND
+       name="show port-channel"
+       help="LAG status and configuration"
+       ptype="SUBCOMMAND"
+       mode="subcommand" />
+    <COMMAND
```

Comment:
Better to add PARAM as summary and keep optional="true", rather than adding every PARAM as separate command.

Reply:
Done here so that summary option selected or not, it is part of command on pressing enter